lib/db.js

```diff
@@ -1,7 +1,8 @@
-
-let Promise = require('bluebird');
 const logger = require('./logger').db;
 let config = require('./config');
+const redisCommands = require('redis-commands');
```

Comment:
@kevinswiber  we already have this dependency, it is not some random module. it is used inside redis js client 
https://github.com/NodeRedis/node_redis/blob/master/lib/commands.js#L3

if we copy this json into the project we will have headache maintaining list up to date with new versions of node-redis